### ISSUE TITLE
(early) timeouts in data extraction request return HTTP/500 instead of 4xx error code

### ISSUE THREAD

**Issue by tyrasd** [GIScience/ohsome-api#10]
When using a relatively short timeout with a data extraction request and the timeout hits before the first chunk of data was sent, the result is a HTTP/500 error code (_internal server error_) instead of a 4xx error code (_invalid request_, e.g. 413 like it is thrown by the data aggregation endpoints).

Also the error description is not very useful in this case:

```bash
$  curl -v "https://api.ohsome.org/v1/elements/geometry?bboxes=-180,-90,180,90&timeout=1"
…
< HTTP/1.1 500 
…
{
  "timestamp" : "2020-06-16T08:49:00.789+0000",
  "status" : 500,
  "error" : "Internal Server Error",
  "message" : "Remote job threw user exception (override or implement ComputeTask.result(..) method if you would like to have automatic 
failover for this exception): null",
  "path" : "/elements/geometry"
}
```

As far as I can see in the logs, the correct `OSHDBTimeoutException` is thrown, but not properly catched and handled by the ohsome API.

**Comment by bonaparten** [comment]
I take a look at it

**Comment by bonaparten** [comment]
I tested it in locale starting the ohsome API in eclipse for the request `http://localhost:8080/elements/geometry?bboxes=-180,-90,180,90&timeout=1` and the error is not reproducible. In my case I get back an 200 status and in the json file an error 413 (which is how it should be).

`{
  "attribution": {
    "url": "https://ohsome.org/copyrights",
    "text": "© OpenStreetMap contributors"
  },
  "apiVersion": "1.0.0-SNAPSHOT",
  "type": "FeatureCollection",
  "features": [
    {
      "timestamp": "2020-07-01T10:07:45.635474",
      "status": 413,
      "message": "The given query is too large in respect to the given timeout. Please use a smaller region and/or coarser time period.",
      "requestUrl": "http://localhost:8080/elements/geometry?bboxes=-180,-90,180,90&timeout=1"
    }`

But this seems to be only the behavior in locale, since the problem is in swagger reproducible.
I tested in swagger smaller bboxes and I got back arbitrarly 500 and 413 errors up to about -3,-1.5,3,1.5

`https://api.ohsome.org/v1/elements/geometry?bboxes=-3,-1.5,3,1.5&timeout=1`
